hydra-node/test/Hydra/Chain/Direct/Contract/Close.hs

```diff
@@ -4,30 +4,16 @@
 
 module Hydra.Chain.Direct.Contract.Close where
 
+import Hydra.Ledger.Cardano hiding (SigningKey)
```

Comment:
I would prefer explicit imports still. At the very least this is a reason for importing `Hydra.Party as Hydra` for a `Hydra.SigningKey`